dalm/pipelines/README.md

```diff
@@ -0,0 +1,16 @@
+## Reading comprehension Pipeline
+
+This is an example of how to string together an end to end reaading-comprehension dataset generator and training regiment in one go
+
+It should be noted that this not the last say, if you the user think there's a better of stringing these together, the components
+are atomic enough for you to string them in other ways that you may deem more appropriate
+
+```bash
+python dalm/pipelines/reading_comprehension_pipeline.py --model_name HuggingFaceH4/zephyr-7b-alpha \
+    --input input.csv --csv_column text \
+	--output_dataset_name combined \
+    --general_spm_path tokenizers/general.spm \
+	--llm_synth_model_name meta-llama/Llama-2-13b \
```

Comment:
Shouldn't this be `meta-llama/Llama-2-13b-chat-hf`?  In order to:

* Use the RLHF'd chat model instead of the raw pretrained model
* Use the HF variation to avoid [errors like this](https://github.com/facebookresearch/llama/issues/394#issuecomment-1641691880)